### PR TITLE
Reader: Limit size of feed search query, truncate query in error notice for feed searches

### LINKED ISSUE #13947: Reader Search on Manage: search errors maybe shouldn't be the same as other Calypso errors
Using the same error box for a search error as when something like a post fails to save seems a bit odd to me. This one in particular is not great given that we are sticking the whole string into the error box.

screen shot 2017-05-11 at 11 29 53 am

props @designsimply

### PR DESCRIPTION
**Pull request by blowery**
Fixes #13947

### CODE REVIEW

**Comment by samouri** [comment]
> Fixes #13947

i don't know about "fixes".  maybe "mitigates"

File: client/state/reader/feed-searches/actions.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ import { READER_FEED_SEARCH_REQUEST, READER_FEED_SEARCH_RECEIVE } from 'state/ac
 
 export const requestFeedSearch = ( query, offset = 0 ) => ( {
 	type: READER_FEED_SEARCH_REQUEST,
-	payload: { query, offset },
+	payload: {
+		query: query.substring( 0, 500 ),
```
Comment (samouri) [inline client/state/reader/feed-searches/actions.js]: shouldn't we be also limiting the input field? or do we want to do this in secret?

Comment (blowery) [inline client/state/reader/feed-searches/actions.js]: sure, we could

Comment (blowery) [inline client/state/reader/feed-searches/actions.js]: done in https://github.com/Automattic/wp-calypso/pull/14286/commits/05264d71c62aaee8b1cfcbe9442c8ee812833ce2

**Comment by samouri** [comment]
Nice! Code LGTM. Testing now

**Pull request merged**